worldengine/draw.py

```diff
@@ -425,8 +395,8 @@ def draw_riversmap_on_file(world, filename):
 
 
 def draw_grayscale_heightmap_on_file(world, filename):
-    img = ImagePixelSetter(world.width, world.height, filename)
-    draw_grayscale_heightmap(world, img)
+    img = PNGWriter.grayscale_from_array(world.elevation['data'], filename, scale_to_range=True)
+    #draw_grayscale_heightmap(world, img)
```

Comment:
I decided to just write out the array. I know this circumvents all modifications usually done in ```draw_grayscale_heightmap()``` but since this file is usable to import data into other programs I didn't like the idea of shifting around elevations.
If this is a bad idea, we can probably add in another parameter to decide which route to take - although I am not sure if a modified heightmap would prove useful at all.